pyproject.toml

```
@@ -2,7 +2,7 @@
 requires = [
      # pin NumPy version used in the build
      "numpy>=1.20.1",
```

```suggestion
     "numpy >= 1.20.1, <2",
```

Just because I noticed it here... we definitely aren't testing numpy 2 yet